index.html

```diff
@@ -31,8 +33,189 @@ <h2 class="support-text">I&rsquo;am a Software Developer</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. h
+            as been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
+        <hrx class="work-cards ">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. 
+            has been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
+        <hrx class="work-cards ">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. 
+            has been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
+        <hrx class="work-cards ">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. 
+            has been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
+        <hrx class="work-cards ">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. 
+            has been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
+        <hrx class="work-cards ">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; 
+            no accounts or sign-ups required. 
+            has been the industry's standard</p>
+          
+          <hrx class="tags">
+            <ul>
+              <li>HTML</li>
+              <li>Bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+          </hrx>    
+          <button type="button">See Project</button>
+        </hrx>
```

Comment:
- [ ] Please make sure your `see project` button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 ![Screenshot from 2022-10-28 21-18-21](https://user-images.githubusercontent.com/92303891/198734579-9bf300cb-7f24-4437-adfe-66a46108e0cf.png)

For instance, 
 
   - You can change the background color, increase the size of the button, and change the color of the text when the user hovers over the button. You can visit [this link](https://www.w3schools.com/css/css3_buttons.asp) to learn more about it. :pray:
  
   - You can give it a `cursor: pointer` property to change the cursor to a pointer when the user hovers over it. You can read more about it [here](https://www.w3schools.com/cssref/pr_class_cursor.asp) :pray: